nuxeo-features/nuxeo-search/nuxeo-search-core/src/main/resources/OSGI-INF/search-pageprovider-contrib.xml

```diff
@@ -121,6 +121,46 @@
       <pageSize>20</pageSize>
     </coreQueryPageProvider>
 
+    <coreQueryPageProvider name="default_trash_search">
+      <trackUsage>true</trackUsage>
+      <whereClause docType="DefaultSearch">
+        <fixedPart>
+          ecm:primaryType NOT IN ('Domain', 'SectionRoot', 'TemplateRoot', 'WorkspaceRoot', 'Favorites')
+          AND ecm:mixinType != 'HiddenInNavigation'
+          AND NOT (ecm:mixinType = 'Collection' AND ecm:name = 'Locally Edited')
```

Comment:
This is going to be very expensive to evaluate at least on SQL. Can't you use another criterion than the facet?

Reply:
`AND NOT (ecm:mixinType = 'Collection' AND ecm:name = 'Locally Edited')` is something used by drive and should be cleaned up soon. And `Collection` facet has `perDocumentQuery="false"` anyway